### DISCUSSION TITLE
Nueva versión v5.2

### DISCUSSION THREAD

**Discussion by jmarcet** [jmarcet/movistar-u7d#33] [Announcements]
- Arreglado problema por cambio absurdo en los datos de la EPG de Movistar.
- Mejorado todo el sistema de detección de la IPTV de Movistar y de la información que se muestra al usuario en caso de problemas.
- Añadido canal "Nick Ukraine".
- Otras pequeñas mejoras.

**Full Changelog**: https://github.com/jmarcet/movistar-u7d/compare/v5.1...V5.2

<hr /><em>This discussion was created from the release <a href='https://github.com/jmarcet/movistar-u7d/releases/tag/v5.2'>Nueva versión v5.2</a>.</em>

**Comment by IvanBaltanas** [comment]
Incluye los canales de deporte? MoVistar LigA de CaMpeones, la LigA, etc.

**Comment by dangerbyte** [comment]
Tengo O2, vale igualmente?
